src/app/registration/facility-select/facility-select.component.ts

```diff
@@ -74,17 +74,23 @@ export class FacilitySelectComponent implements OnInit {
   public stateOrder = ['blank', 'date', 'facility', 'time', 'passes', 'complete'];
   // Initial state
   public state = 0;
+  public systemTime: Date;
+  public systemTimePST: DateTime | null;
+  public allowedToBook: boolean;
+  public validBookingTime = true;
+  public formattedBookingTime: string | null;
 
   constructor(private fb: UntypedFormBuilder, private configService: ConfigService, private toastService: ToastService, private passService: PassService) { }
 
   ngOnInit(): void {
     if (this.configService) {
-      this.trailPassLimit = this.configService.config['TRAIL_PASS_LIMIT'];
+      this.trailPassLimit = this.configService.config['TRAIL_PASS_LIMIT']; 
```

Comment:
No extra space here

Reply:
Resolved in [Commit 43beab5](https://github.com/bcgov/parks-reso-public/pull/461/commits/43beab52a1c7d818910d777c2133aefa47ce60ff)